pkgs/development/python-modules/ki/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, anki
+, cmake
+, python3
```

Comment:
```suggestion
```
inside python-modules the package must be listed individually

Reply:
@SuperSandro2000 is there any one place these sorts of things are documented? im trying to make a checklist of things to look for during reviews and didn't know this one

Reply:
I don't know if it already exists in the documentation and I just missed it, but I only found a reference to this rule in the following release note:

[When adding new or updating current Python libraries, the expressions should be put in separate files in `pkgs/development/python-modules` and called from `python-packages.nix`.](https://github.com/NixOS/nixpkgs/blob/7e151ba98012d774bcc1d1b6a1ccb3a7f94fd865/nixos/doc/manual/release-notes/rl-1703.section.md?plain=1#L293)

As for the requested change, I'm going to resolve the conversation since I'm packaging this tool as an application now.